drivers/video/video_mcux_csi.c

```diff
@@ -443,6 +445,6 @@ static int video_mcux_csi_init_0(const struct device *dev)
 DEVICE_DT_INST_DEFINE(0, &video_mcux_csi_init_0,
 		    NULL, &video_mcux_csi_data_0,
 		    &video_mcux_csi_config_0,
-		    POST_KERNEL, CONFIG_VIDEO_INIT_PRIORITY,
+		    POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE,
```

Comment:
I would add a comment here to highlight the dependency. Otherwise, future folks may not catch the dependency.

Reply:
done